source/guides/basics/custom-items.md

```diff
@@ -1,26 +1,198 @@
 Fancier Sticks: Making Custom Items (PLACEHOLDER ONLY)
 -----------------------------------
 
-**TODO: Write-up that introduces item script containers, and the general ideas of making custom items, mentioning recipes, name/lore, mechanisms, item flags, etc.**
-
-### Placeholder
+```eval_rst
+.. contents:: Table of Contents
+    :local:
+```
 
-Until this page is written, you can view the [old tutorial video here](https://one.denizenscript.com/denizen/vids/Custom%20Items).
+### What Are Custom Items
 
-### Sample Script
+Custom items are a way to store a various set of properties onto an item to create a new unique item in the game. Denizen provides the capabilities to save these unique items in item scripts so that they can be reused or uniquely referenced to in other scripts. These were briefly mentioned in [the mechanisms guide](/guides/basics/mechanisms) but this page will go into more detail.
 
-Here's a quick sample of a modern item script.
+### The Basic Structure
 
+To start off with, this is the basic structure of an item script:
 ```dscript_green
-my_item:
+fancy_stick:
     type: item
     material: stick
     display name: <&b><bold>Fancy stick!
     lore:
     - <&7>So fancy.
+    - <&7>Second line.
 ```
+In [the script editor](/guides/first-steps/script-editor), you can simply type `item` and use tab completion to generate the basic template of an item script at any time.
+
+An item script container has the type set to `item`. You also need to give it a valid existing Minecraft material for your item to be based off of; for example here we are using a stick.
```

Comment:
`You also need to give it a valid existing Minecraft material for your item to be based off of;` this is not actually required. You can put any valid item in the `material:` input, including other item scripts, or even a tag. A default minecraft material is just the most common.